HW05B_script.R

```diff
@@ -0,0 +1,69 @@
+#HW05 Part B
+#Author: Hannah Farrell
+#Date: 8/3/2020
+
+#load appropriate packages
+library(tidyverse)
+library(dplyr)
+library(ggplot2)
+library(stringr)
+
+#load in the data
+AUC1 <- read.csv("./CTRP_files/AUC_1.csv", header = TRUE)
+AUC2 <- read.csv("./CTRP_files/AUC_2.csv", header = TRUE)
+cell_line <- read.csv("./CTRP_files/cancer_cell_line_info.csv", header = TRUE)
+compound <- read.csv("./CTRP_files/compound_info.csv", header = TRUE)
+experiment <- read.csv("./CTRP_files/Experiment_info.csv", header = TRUE)
+
```

Comment:
Did not know "full_join" was also useful to merge tables with the same columns/different rows! (I used bind_rows instead)